source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,21 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	_owner: Bool
-	_forceFree := false
+	_ownsMemory: Bool
 
-	init: func (=_pointer, =_size, owner := false) {
+	init: func (=_pointer, =_size, ownsMemory := false) {
```

Comment:
@davidhesselbom @emilwestergren did you come to an agreement about the default value for `ownsMemory` ?